iTunesSearch/iTunesSearch/ViewController/SearchResultsTableViewController.swift

```diff
@@ -0,0 +1,67 @@
+//
+//  SearchResultsTableViewController.swift
+//  iTunesSearch
+//
+//  Created by Kobe McKee on 5/14/19.
+//  Copyright © 2019 Kobe McKee. All rights reserved.
+//
+
+import UIKit
+
+class SearchResultsTableViewController: UITableViewController, UISearchBarDelegate {
+
+    @IBOutlet weak var mediaSegmentedControl: UISegmentedControl!
+    @IBOutlet weak var searchBar: UISearchBar!
+    
+    let searchResultController = SearchResultController()
+    
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        searchBar.delegate = self
+    }
+
+
+    override func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        return searchResultController.results.count
+    }
+
+    
+    override func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        let cell = tableView.dequeueReusableCell(withIdentifier: "iTunesCell", for: indexPath)
+
+        let mediaItem = searchResultController.results[indexPath.row]
+        cell.textLabel?.text = mediaItem.title
+        cell.detailTextLabel?.text = mediaItem.creator
+        
+        return cell
+    }
+
+
+    func searchBarSearchButtonClicked(_ searchBar: UISearchBar) {
+        guard let searchTerm = searchBar.text else { return }
```

Comment:
It would be a good idea as well to check if the searchTerm contains empty strings. So that we don't let the user submit empty strings to the API 

`guard let searchTerm = searchBar.text,  searchTerm != "" else { return }`